templates/nginx.j2

```diff
@@ -27,6 +27,14 @@ server {
     add_header Strict-Transport-Security "max-age=31536000";
     add_header X-Content-Type-Options nosniff;
 
+    {% for gzip_variable in openwisp2_nginx_ssl_config %}
```

Comment:
you could have named this `config` since this dict can be used also for non-gzip related variables now